WordPress/Classes/ViewRelated/Plans/PlansLoadingIndicatorView.swift

```diff
@@ -0,0 +1,214 @@
+        static let initial: NSTimeInterval  = 0.3
+        static let free: NSTimeInterval     = 0.35
+        static let premium: NSTimeInterval  = 0.0
+        static let business: NSTimeInterval = 0.2
+    }
+
+    // The damping ratio for the spring animation as it approaches its quiescent state.
+    // To smoothly decelerate the animation without oscillation, use a value of 1. Employ a damping ratio closer to zero to increase oscillation.
+    struct SpringDamping {
+        static let free: CGFloat     = 0.5
+        static let premium: CGFloat  = 0.65
+        static let business: CGFloat = 0.5
+    }
+
+    // The initial spring velocity. For smooth start to the animation, match this value to the view’s velocity as it was prior to attachment.
+    // A value of 1 corresponds to the total animation distance traversed in one second. For example, if the total animation distance is 200 points and you want the start of the animation to match a view velocity of 100 pt/s, use a value of 0.5.
+    struct InitialSpringVelocity {
+        static let free: CGFloat     = 0.1
+        static let premium: CGFloat  = 0.01
+        static let business: CGFloat = 0.1
+    }
+
+    struct DefaultSize {
+        static let width: CGFloat    = 100
+        static let height: CGFloat   = 100
+    }
+}
+
+
+// ===========================================================================
+
+
+private extension CGRect {
+    init(center: CGPoint, size: CGSize) {
+        self.origin = CGPoint(x: center.x - size.width / 2, y: center.y - size.height / 2)
+        self.size = size
+    }
+}
+
+private extension CGSize {
+    func scaleBy(scale: CGFloat) -> CGSize {
+        return CGSizeApplyAffineTransform(self, CGAffineTransformMakeScale(scale, scale))
+    }
+}
+
+private extension UIView {
+    var boundsCenter: CGPoint {
+        return CGPoint(x: bounds.midX, y: bounds.midY)
+    }
+}
+
+class PlansLoadingIndicatorView: UIView {
+    private let freeView = UIImageView(image: UIImage(named: "plan-free-loading")!)
+    private let premiumView = UIImageView(image: UIImage(named: "plan-premium-loading")!)
+    private let businessView = UIImageView(image: UIImage(named: "plan-business-loading")!)
+    private let circleView = UIView()
+
+    convenience init() {
+        self.init(frame: CGRect(x: 0, y: 0, width: Config.DefaultSize.width, height: Config.DefaultSize.height))
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = WPStyleGuide.greyLighten30()
+        circleView.clipsToBounds = true
+        circleView.addSubview(premiumView)
+        circleView.addSubview(freeView)
+        circleView.addSubview(businessView)
+        addSubview(circleView)
+
+        freeView.frame = targetFreeFrame
+        premiumView.frame = targetPremiumFrame
+        businessView.frame = targetBusinessFrame
+
+        circleView.backgroundColor = UIColor(red: 211/255, green: 222/255, blue: 230/255, alpha: 1)
```

Comment:
It looks like same color used as the background for the plan icons, so it's probably fine
![screen shot 2016-05-16 at 08 20 06](https://cloud.githubusercontent.com/assets/8739/15281835/04aeec1e-1b3f-11e6-9388-9e74fe07abdf.png)